index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <link rel="stylesheet" href="main.css">
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Document</title>
+</head>
+
+<body>
+ <header>
+  <div class="header-left">
+   <h2>CodePen Home
+    FCC: Tribute Page</h2>
+  </div>
+
+  <div class="header-right">
+   <ul>
+    <li><a>Love</a></li>
+    <li><a>Change View</a></li>
+    <li><a>Sign Up</a></li>
+    <li><a>Log in</a></li>
+   </ul>
+  </div>
+ </header>
+ <div id="main">
+  <h1 id="title">Dr. Norman Borlaug</h1>
+  <h4 id="tribute-info">The man who saved a billion lives</h4>
+  <div id="img-div">
+   <img id="image" src="./image/img.jpg" alt="picture of Dr.Normal Borlaug">
+   <div id="img-caption"> Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat
```

Comment:
I think you overuse the id attribute,I personally would use more class.

Reply:
It is better to add the reason for making a request for a change so that the student understands why he/she has been asked to make the change. Stating why you are suggesting the use of classes over ids here will help the student more than just suggesting it.

Reply:
I think you overuse the id attribute,I personally would use more class.
Id's are unique, they can't be reused,so you should use id only for a unique element of the page.
Classes on the other hand are reusable,you can have a myriad of elements with the same class,and they
all will inherit the CSS rules,thus your CSS file will follow "DRY" approach and its going to be much cleaner.